### ISSUE TITLE
KSCrash 1.8.13 breaks build

### ISSUE THREAD

**Issue by ranunez** [bugsnag/bugsnag-cocoa#119]
The most recent build of KSCrash as of now is 1.8.13, and when building Bugsnag, it throws an error

```
Declaration of 'KSCrashReportFilter' must be imported from module 'KSCrash.KSCrashReportFilter' before it is required
```
<img width="223" alt="screen shot 2016-10-07 at 1 45 50 pm" src="https://cloud.githubusercontent.com/assets/2531425/19205949/45545034-8c9a-11e6-8e05-ed0c1141c4ec.png">

Pinning KSCrash to use 1.8.7 instead works and builds fine.

**Comment by kattrali** [comment]
Thanks for the report, @ranunez. Taking a look.

**Comment by ranunez** [comment]
Some of these commits are highly suspect: https://github.com/kstenerud/KSCrash/commits/master

Might want to reconsider the use of `~>` here
https://github.com/bugsnag/bugsnag-cocoa/blob/master/Bugsnag.podspec.json#L33

**Comment by kattrali** [comment]
@ranunez Yeah, that’s a good call. It seems a bit unstable right now, but I’ve patched the headers and locked the version in 5.6.2.
